cibuild.sh

```diff
@@ -59,6 +59,18 @@ do
     esac
 done
 
+set_msbuild_info()
+{
+    if [ "$OS_NAME" == "Linux" ]; then
+        MSBUILD_ADDITIONALARGS="$MSBUILD_ADDITIONALARGS /p:BaseNuGetRuntimeIdentifier=ubuntu.14.04"
+    elif [ "$OS_NAME" == "Darwin" ]; then
+        MSBUILD_ADDITIONALARGS="$MSBUILD_ADDITIONALARGS /p:BaseNuGetRuntimeIdentifier=osx.10.10"
+    else
+        echo Unrecognized OS $OS_NAME
+        exit 1
+    fi
+}
```

Comment:
Seems unnecessary to spread this logic across multiple files -- couldn't we just `<Exec>` `uname` in the targets file?